File: CoffeeMaker/CSHARP/CoffeeMaker/DryBeansCoffee.cs
using System;
namespace CoffeeMaker
{
    public class DryBeansCoffee : ICoffeeSelect
    {
        public DryBeansCoffee()
        {
        }

        public int GetBeans()
        {
            return 3;
        }

        public bool GrindBeans()
        {
            return true;
        }
    }
}

File: Mocking/CoffeeMaker/Java/src/main/java/DryBeansCoffee.java
import Interfaces.ICoffeeSelect;
import jdk.jshell.spi.ExecutionControl;

public class DryBeansCoffee implements ICoffeeSelect {

    public DryBeansCoffee()
    {
    }

    public int GetBeans() throws ExecutionControl.NotImplementedException {
        throw new ExecutionControl.NotImplementedException("Not implemented");
    }

    public boolean GrindBeans() throws ExecutionControl.NotImplementedException {
        throw new ExecutionControl.NotImplementedException("Not implemented");
    }
}
